src/components/events/eventListingCard.js

```diff
@@ -11,24 +12,49 @@ const Card = styled(Link)`
   color: ${props => props.theme.colors.black};
   overflow: hidden;
   display: flex;
-  flex-direction: column;
   position: relative;
   width: 100%;
+  min-height: 130px;
+
+  ${media.tablet`
+    flex-direction: column;
+    min-height: auto;
+  `};
+`
+const CardImageWrapper = styled.div`
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: cover;
+  background-image: url(${props => props.src});
+  position: relative;
+  overflow: hidden;
+  flex-basis: 40%;
+  flex-shrink: 0;
+  height: 100%;
+
+  ${media.tablet`
+    padding-top: 56.25%;
+  `};
 `
 
 const CardImage = styled.img`
   display: block;
   width: 100%;
+  height: auto;
+  position: absolute;
+  top: 0;
+  left: 0;
+  opacity: 0;
 `
 
 const CardBody = styled.div`
   padding: 20px;
   background-color: ${props => props.theme.colors.white};
   flex-grow: 1;
 
-  @media (min-width: ${props => props.theme.breakpoints[0]}) {
+  ${media.mobile`
```

Comment:
this shouldn't be necessary. Mobile is the default. You can just put `padding: 30px` at the top

Reply:
yes the default is the xs mobile size 0-374px and mobile 375 up. Necessary to cater for devices under 375px like the iphone5 which is 320px

Reply:
Ah, ok. Didn't realise we were using different styles for such small screens 👍

Reply:
There aren't designs for screens under 375, but as a standard 320 is usually the smallest size you build for as the small iphone is that size and quite widely used. When the screen gets down this small it does make sense to reduce things like padding to make the text more readable.